tidy3d/components/device/heat/simulation.py

```diff
@@ -0,0 +1,156 @@
+"""Defines heat simulation class
+NOTE: Keeping this class for backward compatibility only"""
+from __future__ import annotations
+
+from typing import Tuple
+
+from ...types import Ax
+from ...viz import add_ax_if_none, equal_aspect
+from ...scene import Scene
+from ..simulation import DeviceSimulation
+from ....log import log
+
+
+class HeatSimulation(DeviceSimulation):
+    """Contains all information about heat simulation.
+
+    Example
+    -------
+    >>> from tidy3d import Medium, SolidSpec, FluidSpec, UniformUnstructuredGrid, TemperatureMonitor
+    >>> heat_sim = HeatSimulation(
+    ...     size=(3.0, 3.0, 3.0),
+    ...     structures=[
+    ...         Structure(
+    ...             geometry=Box(size=(1, 1, 1), center=(0, 0, 0)),
+    ...             medium=Medium(
+    ...                 permittivity=2.0, heat_spec=SolidSpec(
+    ...                     conductivity=1,
+    ...                     capacity=1,
+    ...                 )
+    ...             ),
+    ...             name="box",
+    ...         ),
+    ...     ],
+    ...     medium=Medium(permittivity=3.0, heat_spec=FluidSpec()),
+    ...     grid_spec=UniformUnstructuredGrid(dl=0.1),
+    ...     sources=[UniformHeatSource(rate=1, structures=["box"])],
+    ...     boundary_spec=[
+    ...         HeatBoundarySpec(
+    ...             placement=StructureBoundary(structure="box"),
+    ...             condition=TemperatureBC(temperature=500),
+    ...         )
+    ...     ],
+    ...     monitors=[TemperatureMonitor(size=(1, 2, 3), name="sample")],
+    ... )
+    """
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        log.warning(
+            "Setting up deprecated ``HeatSimulation``. "
+            "Do consider defining ``DeviceSimulation`` instead."
```

Comment:
for formating code we use double slanted marks `` inside docstrings and single straight ' marks unsure errors, warnings and messages

Reply:
Do we have this documented somewhere (for reference)?